01 - Community/People/Josh Plunkett.md

```diff
@@ -3,25 +3,60 @@ aliases:
 - Joshua Plunkett
 - Sklore
 - Sklex
-tags: 
-- 
+tags:
+- incubator
```

Comment:
What's the thinking behind incubator?

Reply:
Per the tags glossary, it states as such:

```text

- #seedling - All new notes get this tag. If you want to contribute, notes with this tag are usually a great place to start! Might need a template applied to it or if it already has it, no placeholders have been filled out.

- #incubator - Some preliminary work was done, it has a template with placeholders but it was only partially filled out. These notes still have incomplete information and need their placeholders filled out!
```

The differences are clear to me in this case. I would not consider any of the notes I made complete but by the definition I would not consider them seedlings as well. There are still things that can be added.